docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -25,19 +25,22 @@ public class Client {
         HazelcastInstance instance = getHazelcastInstance();
         UserCodeNamespacesConfig parentConfig = instance.getConfig().getNamespacesConfig(); <1>
         UserCodeNamespaceConfig namespaceConfig = new UserCodeNamespaceConfig("my_namespace"); <2>
-        namespaceConfig.addClass(MyRequireClass.class); <3>
-        parentConfig.addNamespaceConfig(namespaceConfig); <4>
+            namespaceConfig.addClass(MyRequireClass.class); <3>
+            UserCodeNamespaceConfig ucn = new UserCodeNamespaceConfig("my_namespace")
+                .addClass("url/to/the/class/file", "classId"); <4>
+        parentConfig.addNamespaceConfig(namespaceConfig); <5>
         // After some time, you can replace `my_namespace` with new resources...
-        UserCodeNamespaceConfig newNamespaceConfig = new UserCodeNamespaceConfig("my_namespace"); <5>
-        newNamespaceConfig.addJar(Paths.get("/etc/path/to/my-resource.jar").toUri().toURL(), "my-resource-jar"); <6>
-        parentConfig.addNamespaceConfig(namespaceConfig); <7>
+        UserCodeNamespaceConfig newNamespaceConfig = new UserCodeNamespaceConfig("my_namespace"); <6>
+        newNamespaceConfig.addJar(Paths.get("/etc/path/to/my-resource.jar").toUri().toURL(), "my-resource-jar"); <7>
+        parentConfig.addNamespaceConfig(namespaceConfig); <8>
     }
 }
 ----
 <1> Fetches the dynamic {ucn} configuration instance from the member
 <2> Creates a new `UserCodeNamespaceConfig` with the name `my_namespace`
 <3> Adds a `Class` resource to the `my_namespace` config, where the `Class` resolved from the client's local context
-<4> Adds the `my_namespace` configuration with its resources to the parent, which is dynamically applied on the member
-<5> Creates a new namespace config with the same name `my_namespace`
-<6> Adds different resources which will replace all previously defined resources
-<7> Adds the new `my_namespace` config to the parent, which will replace the previously defined resources
\ No newline at end of file
+<4> Adds a `Class` resource using the class identifier
```

Comment:
If you do the previous point about adding the class to existing namespace made in point 2, then this will need updated slightly. 

Might be worth saying something like 

"Adds a `Class` resource using the URL to the .class file and a supplied identifier." 


Worth noting a null identifier is valid, in which case the class name will be the ID.